docs/using/syntax.md

```diff
@@ -451,11 +451,16 @@ Target headers are defined with this syntax:
 (header_target)=
 ```
 
-They can then be referred to with the "ref" inline role:
+They can then be referred to with the [ref](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref) inline role:
 
 ```
 {ref}`header_target`
 ```
 
 For example, see this ref: {ref}`syntax/targets` and here's a ref back to the top of
 this page {ref}`example_syntax`.
+
+Alternatively using the markdown syntax; `[](header_target)`, is synonymous with using the [any](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-any) role; `` {any}`header_target` ``
```

Comment:
I did not know that, but that is cool!

Reply:
Indeed, and with the last commit it actually works now lol! I've also added a bit more documentation about using alternative text: https://112-240151150-gh.circle-artifacts.com/0/html/using/syntax.html#targets-and-cross-referencing